src/components/Header.js

```diff
@@ -5,11 +5,12 @@ export default function Header() {
   return (
     <header className={styles.header}>
       <h1>Bookstore CMS</h1>
-      <nav className={styles.navigation}>
+      <nav className={styles.nav}>
         <ul>
-          <li><Link to="/">Books</Link></li>
-          <li><Link to="/categories">Categories</Link></li>
+          <li><Link to="/">BOOKS</Link></li>
+          <li><Link to="/categories">CATEGORIES</Link></li>
         </ul>
+        <img className="icon-button" alt="" />
```

Comment:
Good work adding the icon for a possible user profile button

- Kindly match the Zeplin design, add the missing circle, and position the icon to the right as it is on the Zeplin design (this happens on an upper resolution of 1024px of width)

Your design

![image](https://github.com/aima98/Bookstore/assets/93630700/94b8b40b-6f98-4585-a562-c6862a607a3a)

Zeplin design

![image](https://github.com/aima98/Bookstore/assets/93630700/1c7b0459-82e3-43d1-9434-fbd7e442f152)